esi promises .catch + whitespaces + #53

lethern opened the pull request:
#53 
d9548d5  = #54
#27

Makeshift commented:
My bad, phone didn't show me enough code so I didn't realise this was in a callback.

lethern commented:
commit "handling some of errors"
1.  when user is not in whitelisted alliance, he now gets the message saying so
2. added "setup.permissions.devfleets", if its not set, nothing happens. Helped me in testing things
3. added few errors handling, for null values

Makeshift commented:
Holy jesus you're on fire
